openshift/deploy.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -e
```

Comment:
The script is missing executable bit.

Reply:
@msrb  I ran `chmod +x deploy.sh` to make it executable. Do I need to add anything else in the file.

Reply:
That should do the trick, thanks ;)

Reply:
maybe something like this, which i see in other projects to apply the template
set -x
oc whoami
oc project
set +x

function oc_process_apply() {
  echo -e "\n Processing template - $1 ($2) \n"
  oc process -f $1 $2 | oc apply -f -
}

here=`dirname $0`
template="${here}/template.yaml"

oc_process_apply "$template"

Reply:
I'd also prefer such simpler script. It assumes that developer is already oc-logged on the cluster and has project selected.

But I don't mind if @GeetikaBatra wants to keep this more advanced version of the script :wink: